media-gfx/blender/blender-2.79b.recipe

```diff
@@ -74,13 +74,13 @@ REQUIRES="
 	lib:libopenimageio$secondaryArchSuffix
 #	lib:libopenjp2$secondaryArchSuffix
 	lib:libopenvdb$secondaryArchSuffix
-	lib:libosdCPU$secondaryArchSuffix
-	lib:libosdGPU$secondaryArchSuffix
-	lib:liboslcomp$secondaryArchSuffix
+#	lib:libosdCPU$secondaryArchSuffix
```

Comment:
OSD is broken for 32 bit?

Reply:
I can't install it on 32 bit.

Reply:
There is question mark on x86 in opensubdiv. Binaries are not available in repository.

Reply:
@Begasus : could you please check opensubdiv on 32 bit? Maybe it is just not tested yet.

Reply:
;)

Reply:
On 32bit it fails to detect libglew (even after switching to a local build for glew with cmake for it to create the cmake config files)

Reply:
Almost there with a fix to have this of 32bit :)

Reply:
A PR has been filed for opensubdiv, checking a build for blender on the 32bit Haiku, but as it looks it even fails to detect it even when enabled?
```
-- Could NOT find OpenSubdiv (missing: OPENSUBDIV_INCLUDE_DIR)
```

Reply:
I enabled OpenSubdiv.